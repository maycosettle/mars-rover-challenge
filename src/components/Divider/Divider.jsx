import PropTypes from 'prop-types';

import useStyles from './styles';

const Divider = ({ className }) => {
  const styles = useStyles({ className });

  return <div className={styles.divider} />;
};

Divider.propTypes = {
  className: PropTypes.string,
};

Divider.defaultProps = {
  className: '',
};

export default Divider;
