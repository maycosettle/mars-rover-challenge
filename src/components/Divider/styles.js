import classNames from 'classnames';

const useStyles = ({ className }) => ({
  divider: classNames('w-full border-b border-gray-100', className),
});

export default useStyles;
