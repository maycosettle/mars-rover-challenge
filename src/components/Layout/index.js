export * from './PageLayout';
export * from './PaperLayout';
export * from './PhotoLayout';
