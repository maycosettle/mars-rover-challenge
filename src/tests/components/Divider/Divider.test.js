import { shallow } from 'enzyme';
import { Divider } from 'components/Divider';

describe('tests on <Divider />', () => {
  const wrapper = shallow(<Divider />);

  test('Should render Divider correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
