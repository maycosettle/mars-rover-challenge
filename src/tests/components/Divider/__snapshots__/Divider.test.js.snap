// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <Divider /> Should render Divider correctly 1`] = `
<div
  className="w-full border-b border-gray-100"
/>
`;
